js/packages/mobile-wallet-adapter-protocol/android/build.gradle

```diff
@@ -132,7 +132,7 @@ def kotlin_version = getExtOrDefault('kotlinVersion')
 dependencies {
   //noinspection GradleDynamicVersion
   implementation "com.facebook.react:react-native:+"  // From node_modules
-  implementation "com.solanamobile:mobile-wallet-adapter-clientlib:1.2.0"
+  implementation "com.solanamobile:mobile-wallet-adapter-clientlib:2.0.0-alpha5"
```

Comment:
Updating the underlying native module to use the new clientlib with MWA 2.0 changes